lib/btc-utils.js

```diff
@@ -187,11 +187,16 @@ const waitForBitcoinMempoolToGetTxs = async (btcTxHelper, maxAttempts = 3, check
     return Number(btcToSatoshis(await btcTxHelper.getAddressBalance(btcAddress)));
   };
 
+  const base58AddressToHexString = (base58Address) => {
```

Comment:
`toHexString` doesn't say much. Maybe we should call this `toHash160`? If that is the case, not sure what the hex string is representing here

Reply:
Updated.